pacemaker.spec.in

```diff
@@ -568,13 +568,16 @@ getent group %{gname} >/dev/null || groupadd -r %{gname} -g 189
 getent passwd %{uname} >/dev/null || useradd -r -g %{gname} -u 189 -s /sbin/nologin -c "cluster user" %{uname}
 exit 0
 
-%post -n %{name}-libs -p /sbin/ldconfig
-
-%postun -n %{name}-libs -p /sbin/ldconfig
-
-%post -n %{name}-cluster-libs -p /sbin/ldconfig
+%if %{defined ldconfig_scriptlets}
+%ldconfig_scriptlets libs
+%ldconfig_scriptlets cluster-libs
+%else
+%post libs -p /sbin/ldconfig
+%postun libs -p /sbin/ldconfig
```

Comment:
I don't quite understand the syntax ...

Why did we originally call ldconfig for both library packages' %post but only -libs' %postun? Someone could install/uninstall just the -libs package, or just the -cluster-libs package (if -libs had already been installed), so I would think we would need ldconfig in all situations, to be safe.

Why drop ${name}-?

Why drop -n?

Reply:
It's the same logic across dealing with subpackages with built-in
(directly in spec parser) or native macros, your subpackage
is either of the form:
- `%{name}-foo`, then it's customary to use `%SIGNATURE foo`, or
- `something-else`, then it's customary to use `%SIGNATURE -n something-else`

My guess is this had something to do with expressing that packagers
should preferably keep the subpackages properly namespaced as they
get rewarded with more terse syntax then
(indeed unless new use-case specific packaging conventions apply).

Reply:
Re first question, there are all four variations in the original, perhaps a matter
of how github split the view?

Reply:
Ah, I was looking at this too late in the evening :)

What's here looks good then, let me know when you're done

Reply:
Added reference to documentation at the respective commit.